packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
```

Comment:
![image](https://user-images.githubusercontent.com/93752/61008475-6b8c0780-a324-11e9-86f7-ba13a2804fa4.png)

Reply:
`PropTypes.func` and `PropTypes.string` will do the trick.

Reply:
😬